airgun/widgets.py

```diff
@@ -792,7 +792,8 @@ class ValidationErrors(Widget):
         "not(contains(@style,'display:none'))]"
     )
     ERROR_MESSAGES = (
-        ".//*[(contains(@class,'error-msg-block') "
+        ".//*[(contains(@class, 'alert base in fade alert-danger')"
```

Comment:
Since CSS classes are set, i.e. their order doesn't matter and isn't guaranteed, this looks rather brittle. Next time this alert has classes set to "base alert alert-danger fade in" and XPath won't match it.

I wasn't able to open this particular element in my quick test. How to access it? If I see it, I might have some idea how to better approach it.